fullscreen/api/historical.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<title>Historical Fullscreen features</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+[
+  "onwebkitfullscreenchange",
+  "onwebkitfullscreenerror",
+  "webkitCurrentFullScreenElement",
+  "webkitFullscreenElement",
+  "webkitFullscreenEnabled",
+  "webkitIsFullScreen",
+  "webkitRequestFullScreen",
+  "webkitRequestFullscreen",
+  "webkitDisplayingFullscreen",
+  "webkitEnterFullScreen",
+  "webkitEnterFullscreen",
+  "webkitExitFullScreen",
+  "webkitExitFullscreen",
+  "webkitSupportsFullscreen",
+].forEach(function(member) {
+  ["webkit", "moz", "ms"].forEach(function(prefix) {
+    var alias = member.replace("webkit", prefix);
+    var clarifyTestName = (alias.indexOf('FullScreen') != -1 ? ' (uppercase S)' : '');
+
+    test(function() {
+      assert_false(alias in document.createElement('video'));
+    }, '<video> member must be nuked: ' + alias + clarifyTestName);
```

Comment:
This is efficient and OK with me, but a comment somewhere saying that lots of the things that should be nuked have never existed would be nice.